BUILD.gn

```diff
@@ -251,12 +251,12 @@ if (is_linux) {
 #
 
 # Set ENABLE_PRINTING=1 ENABLE_BASIC_PRINTING=1.
-assert(enable_basic_printing)
-assert(enable_print_preview)
-assert(!enable_service_discovery)
+# assert(enable_basic_printing)
+# assert(enable_print_preview)
+# assert(!enable_service_discovery)
 
 # Enable support for Widevine CDM.
-assert(enable_widevine)
+# assert(enable_widevine)
```

Comment:
The only assert among those that makes sense is `!enable_service_discovery` because CEF has its own implementation that conflicts with the chromium's one at the link step.  The other three look just weird because they want the features to be _enabled_—like if the browser couldn't work without them.

I suggest deleting all of them, WDYT?